fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -368,6 +380,42 @@ public FedoraResource getContainer() {
     }
 
     @Override
+    public FedoraResource getLDPCv() {
+        try {
+            return Optional.of(getNode().getNode(LDPCV_TIME_MAP)).map(nodeConverter::convert).orElse(null);
+        } catch (PathNotFoundException e) {
+            throw new PathNotFoundRuntimeException(e);
+        } catch (RepositoryException e) {
+            throw new RepositoryRuntimeException(e);
+        }
+    }
+
+    @Override
+    public FedoraResource findOrCreateLDPCv() {
+        final Node ldpcvNode;
+        try {
+            ldpcvNode = findOrCreateChild(getNode(), LDPCV_TIME_MAP, NT_FOLDER);
+
+            if (ldpcvNode.isNew()) {
+                LOGGER.debug("Created TimeMap LDPCv {}", node.getPath() + "/" + LDPCV_TIME_MAP);
```

Comment:
It seems like this log message would be less of a maintenance burden by getting the path from: `ldpcvNode.getPath()`.